Sources/LanguageServerProtocol/SupportTypes/SKCompletionOptions.swift

```diff
@@ -18,6 +18,9 @@ public struct SKCompletionOptions: Codable, Hashable {
   /// The maximum number of completion results to return, or `nil` for unlimited.
   public var maxResults: Int?
 
+  @available(*, deprecated, message: "Not used")
```

Comment:
Could you add a the explanation of you PR’s description as a doc comment here?

Reply:
sure. done.